package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@google-cloud/firestore",
   "description": "Firestore Client Library for Node.js",
-  "version": "0.12.0",
+  "version": "0.13.0",
```

Comment:
According to the notes, there are only fixes in this release. Should it be 0.12.1?

Reply:
That's a good question. Technically you're right, but since we had so many issues switching to `grpc` 1.8.x, and since we're jumping through 2 minor `grpc` versions at once, I made it a semver minor. I can still make it a patch release if you think it's more appropriate.

Reply:
I think either way is fine. For consideration, I went with a patch release in the other libs (Bigtable, Datastore, Pub/Sub, possibly others I'm not thinking of at the moment).